tests/test_layers.py

```diff
@@ -193,6 +206,29 @@ def test_tf_squeeze(self):
             expected = model.predict(data)
             self.assertTrue(run_onnx_runtime('onnx_squeeze', onnx_model, data, expected, self.model_files))
 
+    def test_tf_stack(self):
+        def myFunc1(x):
```

Comment:
nit: consistent naming scheme with underscores, call this function test_func_1 or my_func_1